controller_manager/test/test_controller_manager_srvs.cpp

```diff
@@ -188,6 +188,16 @@ TEST_F(TestControllerManagerSrvs, reload_controller_libraries_srv) {
   auto request =
     std::make_shared<controller_manager_msgs::srv::ReloadControllerLibraries::Request>();
 
+  // Create a lambda to store the cleanup state change
+  bool cleanup_called = false;
+  auto cleanup_lambda =
```

Comment:
I would use some explicit name. E.g.,
```suggestion
  auto set_cleanup_called =
```

Reply:
You are right, done.